docs/document/i18n/ja/docusaurus-plugin-content-docs/version-1.4.0/guides/format-setting-plugin.md

```diff
@@ -0,0 +1,416 @@
+---
+id: format-setting-plugin
+title: Format setting plug-in
+sidebar_label: Format setting plug-in
+---
+
+## 概要
+
+kintone のアプリ画面だけでなく、プラグインの設定画面でも kintone UI Component を使うことで活用の幅を広げることができます。<br/>
+今回は簡易的な条件書式プラグインの作成を例に、プラグイン設定画面での kintone UI Component の使い方を説明します。
+
+### 使用するコンポーネント
+- [Button](../components/desktop/button.md)
+- [DatePicker](../components/desktop/date-picker.md)
+- [Dialog](../components/desktop/dialog.md)
+- [Dropdown](../components/desktop/dropdown.md)
+- [MultiChoice](../components/desktop/multichoice.md)
+
+## 完成イメージ
+
+以下が、実装シナリオと画面の完成イメージです。
+
+会社の決算日をプラグイン設定画面で設定し、本日の日付が決算日以前のレコードの特定フィールドを赤くするカスタマイズをプラグイン化します。
+
+![Plugin setting](/img/plugin_setting.png)
+プラグイン設定画面です。<br/>
+設定した条件（本日の日付が基準日付より前、または基準日付より後）を満たすと、設定したフィールドの色がレコード詳細画面で赤色に変わります。
+
+![Confirmation](/img/dialog.png)
+設定を保存する際に、確認ダイアログを表示します。
+
+![App image](/img/payment_management_app.png)
+入金を管理するアプリです。<br/>
+プラグイン設定画面で設定した条件に応じてフィールドの背景色が変わります。
+
+## 事前準備
+
+kintone プラグインの作成にあたり、今回は以下のフォルダ構成で、ファイルを用意します。<br/>
+フォルダ構成、ファイル名などは一例です。
+
+```
+src/
+├── image/
+|   ├── icon.png
+├── js/
+|   ├── config.js
+|   ├── desktop.js
+├── css/
+|   ├── config.css
+├── html/
+|   ├── config.html
+├── lib/
+|   ├── luxon.js
+|   ├── kuc.min.js
+|
+├── manifest.json
+```
+
+## マニフェストファイルの作成
+
+以下のようなマニフェストファイルを作成します。
+
+```json
+{
+  "manifest_version": 1,
+  "version": 1,
+  "type": "APP",
+  "name": {
+    "en": "Format Setting Plug-in"
+  },
+  "description": {
+    "en": "This is a sample plug-in."
+  },
+  "icon": "image/icon.png",
+  "desktop": {
+    "js": [
+      "js/desktop.js",
+      "lib/luxon.js"
+    ],
+    "css": []
+  },
+  "config": {
+    "html": "html/config.html",
+    "js": [
+      "lib/kuc.min.js",
+      "js/config.js"
+    ],
+    "css": [
+      "css/config.css"
+    ],
+    "required_params": ["date", "condition", "targetFields"]
+  }
+}
+```
+
+今回はプラグイン設定画面のカスタマイズで kintone UI Component を使用するので、config オブジェクトの中に UMD ファイルのパスを記載します。<br/>
+UMD ファイルの導入方法は、[Quick Start](../getting-started/quick-start.md) をご覧ください。
+
+
+## HTMLカスタマイズ
+
+プラグイン設定画面用の HTML ファイルを作成します。<br/>
+今回は以下の様に作成します。div 要素に設定した id を利用して、kintone UI Component の各パーツを配置していきます。
```

Comment:
```suggestion
今回は以下のように作成します。div 要素に設定した id を利用して、kintone UI Component の各パーツを配置していきます。
```

Reply:
This paragraph above is used in version 1.0.4 and only gets chnaged from version 1.10.0.
Please double-check
https://kintone-ui-component.netlify.app/docs/ja/1.4.0/guides/format-setting-plugin
https://kintone-ui-component.netlify.app/docs/ja/1.11.0/guides/format-setting-plugin

Reply:
Reply: https://github.com/kintone-labs/kintone-ui-component/pull/1397#discussion_r1278873998

Reply:
Reply: https://github.com/kintone-labs/kintone-ui-component/pull/1397#discussion_r1279265605